api-server/README.md

```diff
@@ -78,6 +69,121 @@ properly configured, with the appropriate .env file and pem file in its home dir
 
 run: `./deploy.sh <environment>`
 
+## Development Environment Setup using Docker & docker-compose
+
+Dependencies:
+- [docker](https://www.docker.com)
+- [casey/just](https://github.com/casey/just/releases/)
+
+### Quick Start
+
+These commands will build local docker images and start the environment.  The
+API can be accessed at http://localhost:3003/api
+
+```console
+$ cp .env.sample .env
+$ cp keys/1nfinite.pem 1nfinite.pem
+$ just build start
+```
+
+Notes:
+* A `.env` file is used to set values for many variables. Multiple tools in the
+  toolchain- `just`, `docker-compose`, and `npm` and `node` (via
+  [dotenv](https://www.npmjs.com/package/dotenv)) all use this file. Different
+  values are used for different enviroments.  The `.env.sample` includes variables
+  appropriate for local development.
+* The key `1nfinite.pem` is used for ??
+* `just` is used as a task runner: the *build* recipe create a docker image and
+  the *start* recipe starts it and a postgres database as docker containers.
+
+Other `just` recipes can be displayed by invoking it without arguments:
+
+```console
+$ just
+Available recipes:
```

Comment:
This all looks pretty cool. I see `start a local docker-based environment via docker-compose`. Does that include the frontend?